app/Models/UserProfileCustomization.php

```diff
@@ -46,9 +46,41 @@ public function setCover($id, $file)
         $this->save();
     }
 
+    /**
+     * An array of all possible profile sections, also in their default order.
+     */
+    public static $sections = ['me', 'recent_activities', 'kudosu', 'top_ranks', 'beatmaps', 'medals', 'historical'];
+
+    public function getExtrasOrderAttribute($value)
+    {
+        return unserialize($value);
```

Comment:
sorry, I keep forgetting about this but this should be stored as json. (just add the column to `$casts` variable above and remove the setter/getter)